plugins/chat/spec/system/shortcuts/chat_composer_spec.rb

```diff
@@ -114,6 +112,8 @@
         find(".chat-composer__input").send_keys(:arrow_up)
 
         expect(page).to have_no_css(".chat-composer-message-details")
+
+        page.driver.browser.network_conditions = { offline: false }
```

Comment:
Cleanup like this should be done in an `ensure` block, so in case of failure state doesn't leak